plugins/Wallet/js/components/sendbutton.js

```diff
@@ -1,13 +1,14 @@
-import React from 'react'
+import React, { PropTypes } from 'react'
 
-const SendButton = ({actions}) => {
-	const handleSendButtonClick = () => actions.startSendPrompt()
-	return (
-		<div onClick={handleSendButtonClick} className="send-button">
-			<i className="fa fa-paper-plane fa-2x"></i>
-			<span>Send Siacoin</span>
-		</div>
-	)
+const SendButton = ({currencytype, onClick}) => (
+	<div onClick={onClick} className="send-button">
+		<i className="fa fa-paper-plane fa-2x"></i>
+		{currencytype === 'siacoins' ? <span>Send Siacoin</span> : <span>Send Siafunds</span>}
```

Comment:
not 100% confident, but  I think I'd prefer the type to be passed directly as the text to render, i.e. `<span>Send {currencyType}</span>`. I don't like branching on magic string values.